Backpack/src/main/res/color/bpk_chip_text_color.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item android:color="@color/bpkWhite" android:state_enabled="true" android:state_selected="true" />
```

Comment:
Just a matter of preference but I prefer to keep this in the code. Won't reject the pr because of this though 😇

Reply:
I try to do `xml` for design/config and code for logic wherever possible.